upm-tools/build-upm-package.sh

```diff
@@ -65,6 +65,10 @@ cp AssemblyDefinitions/BugsnagEditor.asmdef "$PACKAGE_DIR/Editor"
 cp AssemblyDefinitions/BugsnagEditor.asmdef.meta "$PACKAGE_DIR/Editor"
 
 
+# remove EDM menu from package 
+rm "$PACKAGE_DIR/Editor/BugsnagEditor.EDM.cs"
+rm "$PACKAGE_DIR/Editor/BugsnagEditor.EDM.cs.meta"
```

Comment:
Having a `.EDM.cs` file suggests to me that it's a different file type - would it be clearer to put it in an "EDM" subdirectory that can be removed in one command?

```sh
rm -rf "$PACKAGE_DIR/Editor/EDM
```

Reply:
This is the common naming convention for partial classes in Unity. If you feel strongly i'm also ok with changing it, up to you 👍 
Oh and if we put it in a directory we would still need to remove the meta file of that directory, so it would still be 2 commands. Every file and directory in a unity project has a .meta file unfortunatly.

Reply:
Ah - cool, that's fine then.